Makefile

```
@@ -494,6 +495,89 @@ cli-test-snap:
+# Ships as dist/defenseclaw-macos-$(VERSION)-$(GOOS)-$(GOARCH).tar.gz.
+#
+# Overrides: GOOS/GOARCH cross-compile the gateway.
+BUNDLE_GOOS  ?= darwin
+# Universal (x86_64 + arm64 via lipo) is the default for macOS drops so the
+# packaging team ships one artifact for both Intel and Apple Silicon. Override
+# with BUNDLE_GOARCH=amd64 or =arm64 for a single-arch bundle.
+BUNDLE_GOARCH ?= universal
+BUNDLE_NAME  := defenseclaw-macos-$(VERSION)-$(BUNDLE_GOOS)-$(BUNDLE_GOARCH)
+BUNDLE_DIR   := $(DIST_DIR)/$(BUNDLE_NAME)
+
+packaging-macos-bundle:
```

checkmake (0.3.2)

[warning] Target body for "packaging-macos-bundle" exceeds allowed length of 5 lines (50).

(maxbodylength)